src/pybind/mgr/dashboard/controllers/rgw.py

```diff
@@ -77,7 +78,21 @@ def status(self) -> dict:
         except (DashboardException, RequestException, NoRgwDaemonsException) as ex:
             status['message'] = str(ex)  # type: ignore
         return status
-
+    
+@UIRouter('/rgw/multisite')
+class RgwStatus(BaseController):
+    @Endpoint()
+    @ReadPermission
+    def status(self):
+        status = {'available': True, 'message': None}
+        response = json_str_to_object(RgwDaemon.get_multisite_info(self))
```

Comment:
if no multisite is configured, does this returns something? and can we use that info to check rather than checking through `response[0]` and `response[1]`. It looks like a hack.

Reply:
If no multisite is configured, it will send the realm and zonegroup arrays as empty that's why we are checking the response[0] and response [1]